internal/kubernetes/kubernetes.go

```diff
@@ -536,21 +536,15 @@ func (k *Kubernetes) Service(component string, labels ...string) (*corev1.Servic
 		return nil, err
 	}
 
-	selector := []string{
-		fmt.Sprintf("app.infrahq.com/component=%s", component),
-	}
```

Comment:
Doesn't this cause a problem for on-prem when someone does try to deploy them both in the same cluster, because it could match any component (server, ui, etc). Shouldn't this  change to hardcode `connector` as the component here, instead of removing this selector?

Reply:
I actually want to remove this label from the chart. The scenario you're describing is not possible if the server and connector are two separate Helm deployments. I can keep this until that separation is enforced.